skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldDevSettingsEditor.kt

```diff
@@ -16,22 +16,9 @@
 
 package com.google.container.tools.skaffold.run
 
-import com.intellij.execution.configurations.RunConfiguration
-import com.intellij.openapi.options.SettingsEditor
-import javax.swing.JComponent
-import javax.swing.JPanel
-
 /**
  * Settings editor that provides a UI for Skaffold continuous development mode run
  * configuration settings (ultimately calling "skaffold dev"), also saves and retrieves
  * the settings from the project state.
  */
-class SkaffoldDevSettingsEditor : SettingsEditor<RunConfiguration>() {
-    override fun resetEditorFrom(s: RunConfiguration) {
-    }
-
-    override fun createEditor(): JComponent = JPanel()
-
-    override fun applyEditorTo(s: RunConfiguration) {
-    }
-}
+class SkaffoldDevSettingsEditor : BaseSkaffoldSettingsEditor()
```

Comment:
is this still needed now that it has no implementation?

Reply:
We will expand this for single run/continuous run options according to design docs, i.e. tail logs for single run etc.